Userland/Libraries/LibC/fcntl.h

```diff
@@ -32,6 +32,7 @@ int inode_watcher_remove_watch(int fd, int wd);
 int posix_fadvise(int fd, off_t offset, off_t len, int advice);
 int posix_fallocate(int fd, off_t offset, off_t len);
 
+int __utimens(int fd, char const* path, struct timespec const times[2], int flag);
```

Comment:
We don't want to expose this helper to users of LibC do we? It should be file static.

Reply:
Resolved in a discussion in the discord server (https://discord.com/channels/830522505605283862/830525093905170492/1094800326172889129).